src/plugins/discover/server/ui_settings.ts

```diff
@@ -242,15 +243,27 @@ export const getUiSettings: () => Record<string, UiSettingsParams> = () => ({
     category: ['discover'],
     schema: schema.boolean(),
   },
+  [ROW_HEIGHT_OPTION]: {
+    name: i18n.translate('discover.advancedSettings.params.rowHeightTitle', {
+      defaultMessage: 'Maximum lines of document explorer cell',
+    }),
```

Comment:
I'd suggest switching to 
`Displayed lines of document explorer row`
FYI @gchaps